assets/js/phoenix.js

```diff
@@ -851,7 +855,7 @@ export class LongPoll {
     })
   }
 
-  close(code, reason){
```

Comment:
If the intention here was documented passed values, I can add the relevant jsdoc